hw/dv/tools/xcelium/cover.ccf

```diff
@@ -24,6 +24,7 @@ deselect_coverage -betfs -module prim_lfsr...
 
 // Black-box DV CDC module.
 deselect_coverage -betfs -module prim_cdc_rand_delay
```

Comment:
And this applies to both chip and chip_reg_top variants? In VCS they use separate configs...

Reply:
Right - for chip level, we use Xcelium only to run simulations. For analyzing / closing coverage, we are exclusively using VCS.